packages/stores/src/account/utils.ts

```diff
@@ -1,15 +1,28 @@
 import type { Chain } from "@chain-registry/types";
 import { OfflineDirectSigner, OfflineSigner } from "@cosmjs/proto-signing";
+import { StdFee } from "@cosmjs/stargate";
 import {
   ChainName,
   Endpoints,
   ExtendedHttpEndpoint,
   Logger,
 } from "@cosmos-kit/core";
 
+export interface TxFee extends StdFee {
```

Comment:
Can this name be improved, or are you trying to remain generic?

If not, how about something like `EstimateableFee` or similar

Reply:
I'm trying to remain generic. This should be the fee type we use going forward